scanpy/tests/_scripts/seurat_extract_hvg.r

```diff
@@ -0,0 +1,34 @@
+library(reticulate)
```

Comment:
Ah! You're now requiring Seurat in the tests??? That's something that we don't want. No R scripts in the Scanpy tests... The previous version of `filter_genes_dispersion` was benchmarked and the tests should be produced with it; the new function should test against that output.

Seurat might definitely change their function in the future, and we don't want to get in trouble with that. We just want to be consistent within Scanpy.

Reply:
OK, got it; you just used seurat to produce the test data.

Reply:
That's of course fine; it's a bit redundant; but a nice-to-have. :smile:

Reply:
The script can be removed of course, but i thought it would be nice to have in case the dataset changes, for example.

Reply:
yes! it's cool!